modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessorTestCase.java

```diff
@@ -41,6 +47,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class OperationMessageProcessorTestCase extends AbstractOperationMessageProcessorTestCase {
 
+  public static final String INVALID_TARGET =
```

Comment:
this should be the same constant than the validation uses. Make the original constant package level and place this test in the same package